dashboard/test/ui/features/minecraftCodebuilder.feature

```diff
@@ -0,0 +1,91 @@
+Feature: Minecraft CodeBuilder
+
+Scenario: Importing an Agent level from a share link
+  #
+  # Create a new level source
+  #
+  Given I am on "http://studio.code.org/s/hero/stage/1/puzzle/12"
```

Comment:
Should this be a `/s/allthethings` link?

Reply:
I think given that we specifically set up this workflow to work from that actual specific level, it makes sense to have the test point there

Reply:
Ah, although it appears the test is unable to find `/s/hero`. Do we only seed a subset of stages for UI tests?